docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    command: >
+      sh -c "python manage.py wait_for_db &&
+              python manage.py makemigrations &&
```

Comment:
There is no need to include `makemigrations` command, they should be already present in the base code. Remove this line, please